libafl_bolts/src/llmp.rs

```diff
@@ -258,12 +258,20 @@ pub enum TcpRequest {
     LocalClientHello {
         /// The sharedmem description of the connecting client.
         shmem_description: ShMemDescription,
+        /// The `restarter_id` of this client
```

Comment:
We have too many IDs at this point, we should document what it does...
Also, consider a new type (`struct RestarterId(u32)`).